### PR TITLE
Revocation notification pre-AIPv2 changes

### PR DESCRIPTION
**Pull request by smithbk**
Changes as agreed to at the Aries meeting on April 14, 2021 in preparation for AIP v2 to the revocation notification RFC

### CODE REVIEW

**Review by swcurran — APPROVED** [review]
LGTM - thanks @smithbk

File: features/0183-revocation-notification/README.md [hunk 1]
```diff
@@ -42,19 +42,25 @@ The `revoke` message sent by the `issuer` to the `holder` is as follows:
 {
   "@type": "https://didcomm.org/revocation_notification/1.0/revoke",
   "@id": "<uuid-revocation-notification>",
-  "credential_id": "<uuid-credential>",
+  "~please_ack": ["RECEIPT","OUTCOME"],
```
Comment (TimoGlastra) [inline features/0183-revocation-notification/README.md:45]: I updated the please ack protocol in #644. As it stands now the syntax is as follows:
```suggestion
  "~please_ack": {
  	"on": "RECEIPT"
  }
```

**Comment by swcurran** [comment]
@smithbk -- looks like you didn't sign your commit.  Can you please address that so we can merge?

Thanks

**Comment by smithbk** [comment]
Merged under PR 649

**Pull request closed without merging**